App/writefreely

lukasitaly opened the pull request:
[WriteFreely](https://writefreely.org/)

- [x]  Port is not in use already
- [x]  Followed the ["Adding a new app"](https://runtipi.io/docs/contributing/adding-a-new-app) procedure
- [x]  Logo looks good in the app store
- [x] Tested

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Added WriteFreely app to the Tipi App Store, increasing total available apps to 264.
	- Launched a Docker Compose configuration for the WriteFreely service, facilitating easy deployment.

- **Documentation**
	- Updated README.md to include WriteFreely app details.
	- Created a new description.md file outlining WriteFreely’s capabilities and setup process, including admin account creation and key features.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

lukasitaly commented on apps/write-freely/docker-compose.yml:
```diff
@@ -0,0 +1,42 @@
+    ports:
+      - 40:70/tcp
+      - ${APP_PORT}:8080/tcp
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/mnt/config
```
Not needed - see https://hub.docker.com/r/nephatrine/write-freely

lukasitaly commented on apps/write-freely/metadata/description.md:
```diff
@@ -0,0 +1,71 @@
+
+WriteFreely is a clean, minimalist publishing platform made for writers. Start a blog, share knowledge within your organization, or build a community around the shared act of writing.
+
+![](https://i.snap.as/cQZxXoz.png)
```
Done

steveiliop56 commented:
Hello @lukasitaly,

Thank you for the pull request! I am bit confused concerning your port number choices : ). Why did you choose such a big port like 11140 and not something like 8574. Also I see that the container is using port 70 too, a quick google search says that is for a protocol called the gohper protocol, maybe we should keep port 70 instead of 40? Apart from these everything looks ok and the app works so we can merge.

lukasitaly commented:
Hey @steveiliop56, thanks for the comment!

I didn't realize I was using port 40, and I'm not sure why I changed it. I had been using the same YAML file up until now, so I just copied it over. I also chose port 11140 for the same reason - I prefer using high ports because they're less likely to be used by other services or Runtipi. Since I checked and saw that another app in the Tipi store wasn't using it, I decided to stick with it.

I changed the ports as you suggested and now everything should be in order :)

steveiliop56 commented:
Cool! I am merging!
